### DISCUSSION TITLE
Allow experimental 'core/table-of-contents'

### DISCUSSION THREAD

**Discussion by demo-igor** [WordPress/gutenberg#50223] [General]
Is it possible to allow in Wordpress experimental 'core/table-of-contents' ?
I see that config for this block has `__experimental: true` prop.
I can to make it `false` in the core WP code, but want to find more smarter way.

**Comment by demo-igor** [comment]
@adamziel could you help to understand is it possible or not to enable experimental blocks in WordPress?

**Reply by adamziel** [reply]
@demo-igor The experimental blocks are restricted to the Gutenberg plugin and removed before merging the code to WordPress core. You should be able to access the block through the plugin, but it will only be available in core once the experimental flag is removed. As to when will that be, this discussion might clear that up: https://github.com/WordPress/gutenberg/issues/42229

**Comment by t-hamano** [comment]
My understanding is that blocks marked as `__experimental: true` in `block.json` should not be bundled with the WP core itself.
Did you change `__experimental` to `false` in the Gutenberg plugin?

As a reference, there is #42229 to consider stabilizing this block.

**Reply by demo-igor** [reply]
I see that is that blocks marked as `__experimental: true` bundled in the WP core. And I can modify bundle and make experimental block works but in this way I should handle update WP core and make this change every time. It will be really cool to have some filter/action to allow experimental blocks if it needs without updating minimized bundle. Maybe we already have it but I din't find for now
